assets/javascripts/discourse/components/topic-chat-float.js

```diff
@@ -256,10 +261,6 @@ export default Component.extend({
 
     // Set activeChannel to null to avoid a moment where the chat composer is rendered twice.
     // Since the mobile-file-upload button has an ID, a JS error will break things otherwise.
```

Comment:
That comment seems to be out of place after these changes. Should it be moved or updated?

Reply:
I will just remove it I think, it doesn’t make much sense anymore, thanks.